dvc/fs/ssh/__init__.py

```diff
@@ -119,16 +119,16 @@ def ensure_credentials(self):
         if self.ask_password and self.password is None:
             self.password = ask_password(self.host, self.user, self.port)
 
-    def ssh(self, path_info):
+    def ssh(self):
```

Comment:
One issue that arises here is that `move()` in `ObjectDB.add()` will no longer work, as we assume that src path and object path belong to the same fs, which they don't. This is a common problem for other filesystems too in external-output scenario, so I wound't fix it right now, as that is a hack already that we are keeping https://github.com/iterative/dvc/blob/master/dvc/objects/db/base.py#L78 for regular non-extrnal `dvc add` and such. We will be adjusting that in save/transfer unification anyway.

Reply:
Also, now that this doesn't take any arguments, we could adjust the way it looks and works, but again I would just wait for `sshfs` migration, since it will simply throw away this code anyway, so doesn't seem like it is worth wasting time refactoring this now.